test.sh

```diff
@@ -3,7 +3,7 @@
 #
 # With singularity on ollie
 #
-# module load singularity/3.5.1
+# module load singularity/3.9.1
```

Comment:
Oh, that's nice! :)

Reply:
We may automate that to build newest singularity on Ollie whenever they make a release. But more likely not until albedo